7.3/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+php:
+    build: .
+    working_dir: /var/www/app
+    environment:
+        PHP_FPM_USER: root
+        PHP_FPM_PORT: 9000
+        PHP_ERROR_REPORTING: E_ALL
+        ENVIRONMENT: staging
```

Comment:
Can't remember why we set staging as default in the `docker-compose.yml` file. 🤷‍♂️